OHIF/Viewers
# fix various bugs

### Context & Changes & Results

- improve the UX by showing a notif for non TID1500 SRs
- Fix creation of the segmentation for the TMTV
- Fix local modalities
- Fix docs



### Testing

Fix creation of the segmentation for the TMTV
- Go to tmtv, create a new Label -> it fails 

Fix local modalities
- Drag and drop in local, the study browser doesn't show modalities

![image](https://github.com/OHIF/Viewers/assets/7490180/01c06e5c-4b47-4a9e-a722-6d75d50a81bf)

Docs 
Right now there is some weird collapse behaviour for v3-docs.ohif.org, 
- Go to link
- go to configuration > datasources 
- see not all appearing 




### Checklist

#### PR



- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates



- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] "OS: 
- [] "Node version: 
- [] "Browser:
  


[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch
